test/rules/increment-decrement/default/test.ts.lint

```diff
@@ -0,0 +1,20 @@
+let x = 7;
+
+++x;
+~~~       [plus]
```

Comment:
There are actually some edge cases when fixing this:

```ts
a + b++;
// currently becomes
a + b += 1; // which is invalid
// should become
a + (b += 1);
```

Reply:
please fix this.